src/OrchardCore/OrchardCore/Extensions/Features/FeatureHash.cs

```diff
@@ -30,7 +31,7 @@ public async Task<int> GetFeatureHashAsync()
 
             serial = enabledFeatures
                 .OrderBy(x => x.Id)
-                .Aggregate(0, (a, f) => a * 7 + f.Id.GetHashCode());
+                .Aggregate(0, (a, f) => HashCode.Combine(a, f.Id.GetHashCode()));
```

Comment:
Should be something like this:





var hc = new HashCode();

foreach -> hc.Add(f.Id)

serial = hc.ToHashCode()

Reply:
Where is `Hash.Combine()` then?

Feel free to close this and start new PR

Reply:
`Hash.Combine` is a static helper for a specific number of items, here the items in unbound, so we use `.Add()`